tests/rptest/services/redpanda.py

```diff
@@ -91,14 +97,17 @@ def start_node(self, node, override_cfg_params=None):
 
         self.write_conf_file(node, override_cfg_params)
 
+        if self.coproc_enabled():
+            self.start_wasm_engine(node)
```

Comment:
do you think we should run ducktape tests with coproc always enabled?

Reply:
Only if redpanda is started with the `enable_coproc` options. Thats the way its done here.